cmd/antrea-agent/options.go

```diff
@@ -192,17 +192,45 @@ func (o *Options) setDefaults() {
 func (o *Options) validateFlowExporterConfig() error {
 	if features.DefaultFeatureGate.Enabled(features.FlowExporter) {
 		if o.config.FlowCollectorAddr == "" {
-			return fmt.Errorf("IPFIX flow collector address should be provided")
+			// If empty string is given, we initialize o.flowCollector with fields
+			// to connect to the flow aggregator service. We consider TCP as default
+			// transport for flow aggregator.
+			o.flowCollector = &net.TCPAddr{
```

Comment:
It seems that this is more verbose that it needs to be: if you omit initialization of the `IP` and `Zone` field, these default values will be used